PurchasesTests/LocalReceiptParsing/Builders/InAppPurchaseBuilderTests.swift

```diff
@@ -261,67 +268,93 @@ private extension InAppPurchaseBuilderTests {
     }
 
     func quantityContainer() -> ASN1Container {
-        return containerFactory.receiptAttributeContainer(attributeType: InAppPurchaseAttributeType.quantity,
-                                                               quantity)
+        containerFactory.receiptAttributeContainer(
+            attributeType: InAppPurchaseAttributeType.quantity,
+            quantity
+        )
     }
 
     func productIdContainer() -> ASN1Container {
-        return containerFactory.receiptAttributeContainer(attributeType: InAppPurchaseAttributeType.productId,
-                                                               productId)
+        containerFactory.receiptAttributeContainer(
+            attributeType: InAppPurchaseAttributeType.productId,
+            productId
+        )
     }
 
     func transactionIdContainer() -> ASN1Container {
-        return containerFactory.receiptAttributeContainer(attributeType: InAppPurchaseAttributeType.transactionId,
-                                                               transactionId)
+        containerFactory.receiptAttributeContainer(
+            attributeType: InAppPurchaseAttributeType.transactionId,
+            transactionId
+        )
     }
 
     func originalTransactionIdContainer() -> ASN1Container {
-        return containerFactory.receiptAttributeContainer(attributeType: InAppPurchaseAttributeType.originalTransactionId,
-                                                               originalTransactionId)
+        containerFactory.receiptAttributeContainer(
```

Comment:
I'd leave the `return`s in all these.

Reply:
yep, no problem. Just to know, is there a reason for that? 🤔

Reply:
It's a matter of preference. I think it makes sense for simple `var`s, but for `func`s like this it almost seems like it's not returning anything. `var`s HAVE to `return` something, but `func`tions don't, so I think being explicit makes sense.

Reply:
good point. Thanks for the clarification